pkg/docker/docker.go

```diff
@@ -132,6 +131,72 @@ type RunContainerOptions struct {
 	CapDrop          []string
 }
 
+// asDockerConfig converts a RunContainerOptions into a Config understood by the
+// go-dockerclient library.
+func (rco RunContainerOptions) asDockerConfig() docker.Config {
+	return docker.Config{
+		Image: getImageName(rco.Image),
+		User:  rco.User,
+		Env:   rco.Env,
+
+		// TODO review how callers use these options to pass stdin.
+		// Probably they are not necessary, as they're related to
+		// attaching to the container (that is often unnecessary).
```

Comment:
These options here should be the same as they were [before](https://github.com/openshift/source-to-image/pull/430/files#diff-bf07e8f23e3aaca43157f262317e094cL593).

Stderr streaming is setup in [line 189](https://github.com/openshift/source-to-image/pull/430/files#diff-bf07e8f23e3aaca43157f262317e094cR189) and [193](https://github.com/openshift/source-to-image/pull/430/files#diff-bf07e8f23e3aaca43157f262317e094cR193).